packages/clerk-js/src/core/resources/UserOrganizationInvitation.ts

```diff
@@ -29,10 +30,22 @@ export class UserOrganizationInvitation extends BaseResource implements UserOrga
   static async retrieve(
     params?: GetUserOrganizationInvitationsParams,
   ): Promise<ClerkPaginatedResponse<UserOrganizationInvitation>> {
+    function convertPageToOffset(pageParams: GetUserOrganizationInvitationsParams | undefined): ClerkPaginationParams {
```

Comment:
🔧 This function can be a separate module as I assume it will be required in more resources soon.